doc/changes/changes_1.2.1.md

```diff
@@ -0,0 +1,33 @@
+# error-code-crawler-maven-plugin 1.2.1, released 2022-10-24
+
+Code name: Fix vulnerabilities in dependencies
+
+## Summary
+
+This release fixes CVE-2022-42003 and CVE-2022-42004 in `com.fasterxml.jackson.core:jackson-databind`.
+
+## Bugfixes
+
+* #81: Fixed vulnerabilities in dependencies
+
+## Dependency Updates
+
+### Compile Dependency Updates
+
+* Updated `com.exasol:error-code-model-java:2.1.0` to `2.1.1`
+* Updated `com.exasol:error-reporting-java:0.4.1` to `1.0.0`
+* Updated `com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.13.3` to `2.13.4`
```

Comment:
:heavy_check_mark: I checked the CVE list. 2.13.4 fixes CVE-2022-42003 and CVE-2022-42004.

See: https://cve.report/software/fasterxml/jackson-databind